frontend/src/scenes/plugins/edit/PluginSource.tsx

```diff
@@ -101,9 +102,18 @@ export function PluginSource(): JSX.Element {
                 {editingSource ? (
                     <>
                         <p>
-                            <a href="https://posthog.com/docs/plugins/build" target="_blank">
+                            Feeling lost?{' '}
```

Comment:
Hmm, this question suggests a bit that this is a common feeling with this UX. 😅 Would rather go for a _statement_ here, in the spirit of "BTW you can check that one page out if you want to know more"